lib/jekyll-feed/feed.xml

```diff
@@ -56,10 +56,18 @@
       {% endunless %}
 
       {% assign post_author = post.author | default: post.authors[0] | default: site.author %}
+      {% assign post_author_name = nil %}
+      {% unless post_author.first %}
+        {% assign post_author_name = post_author %}
+      {% endunless %}
       {% assign post_author = site.data.authors[post_author] | default: post_author %}
       {% assign post_author_email = post_author.email | default: nil %}
       {% assign post_author_uri = post_author.uri | default: nil %}
-      {% assign post_author_name = post_author.name | default: post_author %}
+      {% assign post_author_twitter = post_author.twitter | default: nil %}
+      {% if post_author_twitter %}
+        {% assign post_author_twitter = post_author.twitter | prepend: "@" %}
+      {% endif %}
```

Comment:
I don't understand why this logic is so complex.
Isn't the following equivalent (to ln66 - ln69):
```
{% if post_author.twitter %}
  {% assign post_author_twitter = post_author.twitter | prepend: "@" %}
{% else %}
  {% assign post_author_twitter = nil %}
{% endif %}
```

Reply:
@ashmaroli You are right, I didn't thought about it.
The outer if block was added after I figure out the prepend filter break the nil check. I will update it later.

Reply:
Thanks. While you update it, please update the logic for `post_author_name` as well.